### PR TITLE
Wrote and updated all charge descriptions 

### PR DESCRIPTION
**Pull request by Crayons0814**
Hopefully these are spelled correctly. DISCLAIMER these are more like place-holders to give an idea of what each charge could mean, it is HIGHLY recommended to write your own.

### CODE REVIEW

**Comment by LeSiiN** [comment]
Would be a shame to not add that, so much work have been done

File: shared/config.lua [hunk 1]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [3] = {title = 'Aggravated Assault', class = 'Felony', id = 'P.C. 1003', months = 20, fine = 1250, color = 'orange', description = 'When a person unintentionally, and recklessly causes bodily injury to another as a result of a confrontation AND causes bodily injury'},
+        [4] = {title = 'Assault with a Deadly Weapon', class = 'Felony', id = 'P.C. 1004', months = 30, fine = 3750, color = 'red', description = 'When a person intentionally, knowingly, or recklessly causes bodily injury to another person AND either causes serious bodily injury or uses or exhibits a deadly weapon'},
+        [5] = {title = 'Involuntary Manslaughter', class = 'Felony', id = 'P.C. 1005', months = 60, fine = 7500, color = 'red', description = 'When a person unintentionally and recklessly causes the death of another'},
+        [6] = {title = 'Vehicular Manslaughter', class = 'Felony', id = 'P.C. 1006', months = 75, fine = 7500, color = 'red', description = 'when a person unintentionally and recklessly causes the death of anther with a vehicle'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Capital W

File: shared/config.lua [hunk 2]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
-        [22] = {title = 'Criminal Threats', class = 'Misdemeanor', id = 'P.C. 1022', months = 5, fine = 500, color = 'orange', description = 'Insert Charge Description Here'},
-        [23] = {title = 'Reckless Endangerment', class = 'Misdemeanor', id = 'P.C. 1023', months = 10, fine = 1000, color = 'orange', description = 'Insert Charge Description Here'},
-        [24] = {title = 'Gang Related Shooting', class = 'Felony', id = 'P.C. 1024', months = 30, fine = 2500, color = 'red', description = 'Insert Charge Description Here'},
-        [25] = {title = 'Cannibalism', class = 'Felony', id = 'P.C. 1025', months = 0, fine = 0, color = 'red', description = 'Insert Charge Description Here'},
-        [26] = {title = 'Torture', class = 'Felony', id = 'P.C. 1026', months = 40, fine = 4500, color = 'red', description = 'Insert Charge Description Here'},
+        [1] = {title = 'Simple Assault', class = 'Misdemeanor', id = 'P.C. 1001', months = 7, fine = 500, color = 'green', description = 'when a person intentionally or knowingly causes physical contact with another (without a weapon)'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Capital W

File: shared/config.lua [hunk 3]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
-        [8] = {title = 'Forgery', class = 'Misdemeanor', id = 'P.C. 3008', months = 15, fine = 750, color = 'green', description = 'Insert Charge Description Here'},
-        [9] = {title = 'Money Laundering', class = 'Felony', id = 'P.C. 3009', months = 0, fine = 0, color = 'red', description = 'Insert Charge Description Here'},
+        [1] = {title = 'Impersonating', class = 'Misdemeanor', id = 'P.C. 3001', months = 15, fine = 1250, color = 'green', description = 'The action of falsely identifying as another person to deceive'},
+        [2] = {title = 'Impersonating a Peace Officer or Public Servant', class = 'Felony', id = 'P.C. 3002', months = 25, fine = 2750, color = 'green', description = 'The action of falsely identifying as a government employee to deceive'},
+        [3] = {title = 'Impersonating a Judge', class = 'Felony', id = 'P.C. 3003', months = 0, fine = 0, color = 'green', description = 'The action of falsely identifying as a Judge to deceive'},
+        [4] = {title = 'Possession of Stolen Identification', class = 'Misdemeanor', id = 'P.C. 3004', months = 10, fine = 750, color = 'green', description = 'To Have another persons Identification without consent'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Remove capital H

Comment (MonkeyWhisper) [reply shared/config.lua]: and remove capital I, identification

File: shared/config.lua [hunk 4]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [5] = {title = 'Possession of Stolen Government Identification', class = 'Misdemeanor', id = 'P.C. 3005', months = 20, fine = 2000, color = 'green', description = 'To have the Identification of a Government employee without consent'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: identification and government, remove capital I and G

File: shared/config.lua [hunk 5]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [3] = {title = 'Reckless Evading', class = 'Felony', id = 'P.C. 9003', months = 10, fine = 800, color = 'orange', description = 'Recklessly disregarding safety and Hiding or running from lawful detainment while '},
+        [4] = {title = 'Failure to Yield to Emergency Vehicle', class = 'Infraction', id = 'P.C. 9004', months = 0, fine = 600, color = 'green', description = 'Not giving way to emergency vehicles'},
+        [5] = {title = 'Failure to Obey Traffic Control Device', class = 'Infraction', id = 'P.C. 9005', months = 0, fine = 150, color = 'green', description = 'Not following the safety devices of the roadway'},
+        [6] = {title = 'Nonfunctional Vehicle', class = 'Infraction', id = 'P.C. 9006', months = 0, fine = 75, color = 'green', description = 'Having a vehicle that is no longer functional in the roadway'},
+        [7] = {title = 'Negligent Driving', class = 'Infraction', id = 'P.C. 9007', months = 0, fine = 300, color = 'green', description = 'Driveing in a manner as to unknowingly disregard safety'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Typo here, Driving

File: shared/config.lua [hunk 6]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [8] = {title = 'Reckless Driving', class = 'Misdemeanor', id = 'P.C. 9008', months = 10, fine = 750, color = 'orange', description = 'Driveing in a manner as to knowingly disregard safety'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Typo here, Driving

File: shared/config.lua [hunk 7]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [9] = {title = 'Third Degree Speeding', class = 'Infraction', id = 'P.C. 9009', months = 0, fine = 225, color = 'green', description = 'Speeding 15 over the limit'},
+        [10] = {title = 'Second Degree Speeding', class = 'Infraction', id = 'P.C. 9010', months = 0, fine = 450, color = 'green', description = 'Speeding 35 over the limit'},
+        [11] = {title = 'First Degree Speeding', class = 'Infraction', id = 'P.C. 9011', months = 0, fine = 750, color = 'green', description = 'Speeding 50 over the limit'},
+        [12] = {title = 'Unlicensed Operation of Vehicle', class = 'Infraction', id = 'P.C. 9012', months = 0, fine = 500, color = 'green', description = 'The operation of a motor vehicle without proper licensing'},
+        [13] = {title = 'Illegal U-Turn', class = 'Infraction', id = 'P.C. 9013', months = 0, fine = 75, color = 'green', description = 'Performing a u-turn where it is prohibited'},
+        [14] = {title = 'Illegal Passing', class = 'Infraction', id = 'P.C. 9014', months = 0, fine = 300, color = 'green', description = 'passing other motor vehicles in a prohibited manner'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Capital P, passing

File: shared/config.lua [hunk 8]
```diff
@@ -98,205 +98,205 @@ Config.PenalCodeTitles = {
...
+        [15] = {title = 'Failure to Maintain Lane', class = 'Infraction', id = 'P.C. 9015', months = 0, fine = 300, color = 'green', description = 'Not staying in the correct lane with a motor vehicle'},
+        [16] = {title = 'Illegal Turn', class = 'Infraction', id = 'P.C. 9016', months = 0, fine = 150, color = 'green', description = 'performing a turn where it is prohibited'},
```
Comment (MonkeyWhisper) [inline shared/config.lua]: Capital P, performing

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
This is what you meant by that, I thought that suggestion was for more charges. 

Great job, I bet this took quite some time, thank you for that, we appreciate it.

I did glance over it and found some small typos and whatnot, other than that, very good!

**Comment by Crayons0814** [comment]
Should be good now

**Comment by MonkeyWhisper** [comment]
Great job! Thanks

**Pull request merged**